monitoring/prometheus-operator.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   chart:
     repository: https://kubernetes-charts.storage.googleapis.com/
     name: prometheus-operator
-    version: 5.1.0
+    version: 9.3.1
```

Comment:
This was fixed by https://github.com/weaveworks/eks-quickstart-app-dev/pull/40/files#diff-4687e5b7b80ab2d35494da64524dad76R12